tools/fs/fsFixPath.ts

```diff
@@ -0,0 +1,43 @@
+export {
+  // The tools/fs/files module used to export wrappers for both fiberized
+  // and synchronous fs.* functions. This module exists to preserve backwards
+  // compatibility with that behavior, even though everything is sync now.
+  appendFile as appendFile,
+  appendFile as appendFileSync,
+  chmod as chmod,
+  chmod as chmodSync,
+  close as close,
+  close as closeSync,
+  createReadStream as createReadStream,
+  createWriteStream as createWriteStream,
+  lstat as lstat,
+  lstat as lstatSync,
+  mkdir as mkdir,
+  mkdir as mkdirSync,
+  open as open,
+  open as openSync,
+  read as read,
+  readFile as readFile,
+  readFile as readFileSync,
+  read as readSync,
+  readdir as readdir,
+  readdir as readdirSync,
+  readlink as readlink,
+  readlink as readlinkSync,
+  realpath as realpath,
+  realpath as realpathSync,
+  rmdir as rmdir,
+  rmdir as rmdirSync,
+  stat as stat,
+  stat as statSync,
+  symlink as symlink,
+  symlink as symlinkSync,
+  unlink as unlink,
+  unlink as unlinkSync,
+  unwatchFile as unwatchFile,
+  watchFile as watchFile,
+  write as write,
+  writeFile as writeFile,
+  writeFile as writeFileSync,
+  write as writeSync,
+} from "./files";
```

Comment:
One beautiful thing about the [Reify](https://github.com/benjamn/reify) module compiler is that all this repetition becomes just
```js
module.link("./files", {
  appendFile: ["appendFile", "appendFileSync"],
  chmod: ["chmod", "chmodSync"],
  close: ["close", "closeSync"],
  createReadStream: "createReadStream",
  createWriteStream: "createWriteStream",
  lstat: ["lstat", "lstatSync"],
  mkdir: ["mkdir", "mkdirSync"],
  open: ["open", "openSync"],
  read: ["read", "readSync"],
  readFile: ["readFile", "readFileSync"],
  readdir: ["readdir", "readdirSync"],
  readlink: ["readlink", "readlinkSync"],
  realpath: ["realpath", "realpathSync"],
  rmdir: ["rmdir", "rmdirSync"],
  stat: ["stat", "statSync"],
  symlink: ["symlink", "symlinkSync"],
  unlink: ["unlink", "unlinkSync"],
  unwatchFile: "unwatchFile",
  watchFile: "watchFile",
  write: ["write", "writeSync"],
  writeFile: ["writeFile", "writeFileSync"]
}, 0);
```